cmd/root.go

```diff
@@ -218,3 +218,9 @@ func logInvocationParams(cmd *cobra.Command, args []string) {
 
 	l.Str("command", cmd.CommandPath()).Strs("args", args).Msg("Invocation parameters")
 }
+
+func initReattachMode() {
+	if viper.GetString("reattach-providers") != "" {
```

Comment:
Unrelated to the fix, but I don't think there's a case we'd want to verify when `reattach-providers` is set (correct me if I'm wrong).
When verifying we download the checksum from GitHub, so not sure how that can work with a local provider (unless the user ensures those match).

Reply:
that is correct, but also incorrect, if I have one provider (aws) not reattached and one provider in reattach (datadog) if we disable verify than aws provider is not verified.

the plugin manager should not verify/download plugins that are reattached, so I would add the fix in the pluginmanager to avoid this check for reattached providers.